packages/Webkul/Category/src/Repositories/CategoryRepository.php

```diff
@@ -50,7 +50,13 @@ public function getAll(array $params = [])
 
                     break;
                 case 'parent_id':
-                    $queryBuilder->where('categories.parent_id', $value);
+                    // Check if the value contains commas (multiple parent IDs)
+                    if (strpos($value, ',') !== false) {
+                        $parentIds = array_map('trim', explode(',', $value));
+                        $queryBuilder->whereIn('categories.parent_id', $parentIds);
+                    } else {
+                        $queryBuilder->where('categories.parent_id', $value);
+                    }                    
```

Comment:
I think this case can be handled more cleanly using this approach.

~~~php
$parentIds = array_filter(array_map('trim', explode(',', $value)));

$queryBuilder->whereIn('categories.parent_id', $parentIds);
~~~

Reply:
Applied the suggestion in [this commit](https://github.com/bagisto/bagisto/pull/10614/commits/09a0af33bae670df1ee0be0a8eb814905053e522).